ui/src/components/member/AddPoc.js

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright The Athenz Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import AddModal from '../modal/AddModal';
+import MemberUtils from '../utils/MemberUtils';
+import styled from '@emotion/styled';
+import InputLabel from '../denali/InputLabel';
+import InputDropdown from '../denali/InputDropdown';
+import { colors } from '../denali/styles';
+import { selectAllUsers } from '../../redux/selectors/user';
+import { USER_DOMAIN } from '../constants/constants';
+import RequestUtils from '../utils/RequestUtils';
+
+const SectionsDiv = styled.div`
+    width: 760px;
+    text-align: left;
+    background-color: ${colors.white};
+`;
+
+const SectionDiv = styled.div`
+    align-items: flex-start;
+    display: flex;
+    flex-flow: row nowrap;
+    padding: 10px 30px;
+`;
+
+const ContentDiv = styled.div`
+    flex: 1 1;
+    display: flex;
+    flex-flow: row nowrap;
+`;
+
+const StyledInputLabel = styled(InputLabel)`
+    flex-basis: 20%;
+    font-weight: 600;
+    line-height: 36px;
+`;
+
+const StyledInput = styled(InputDropdown)`
+    flex-basis: 80%;
+`;
+
+class AddPoc extends React.Component {
+    constructor(props) {
+        super(props);
+        this.api = props.api;
+        this.onSubmit = this.onSubmit.bind(this);
+        this.userSearch = this.userSearch.bind(this);
+        this.state = {
+            showModal: this.props.isOpen,
+            showAddPoc: false,
+            pocName: '',
+        };
+    }
+
+    onSubmit() {
+        if (!this.state.pocName || this.state.pocName.trim() === '') {
+            this.setState({
+                errorMessage: 'Point of Contact is required for Update.',
+            });
+            return;
+        }
+        let meta = {
+            contacts: this.props.contacts,
+        };
+        meta.contacts[this.props.contactType] = this.state.pocName;
+        let domainName = this.props.domain;
+        let auditMsg =
+            'Updating Point of Contact ' +
+            this.state.pocName +
+            ' for domain ' +
+            this.props.domain;
```

Comment:
Maybe consider using `Updating Point of Contact ${this.state.pocName} for domain ${this.props.domain}`?

Reply:
good suggestion, fixed